README.es.md

```diff
@@ -0,0 +1,440 @@
+  revisoras/es**.
+
+- La naturaleza abierta de nuestro proceso de revisión te permite
+  **establecer redes y conocer colegas y personas que colaboran**.
+  Nuestra comunidad es amigable y está llena de miembros con
+  conocimiento en el desarrollo de R y en muchas otras áreas de la
+  ciencia y la informática científica con ganas de ayudar.
+
+Para ofrecerte para revisar paquetes, completa [este breve
+formulario](https://ropensci.org/software-reviewer/) proporcionando tu
+información de contacto y tus áreas de experiencia. Siempre buscamos más
+personas con experiencia en el desarrollo de paquetes en general y con
+conocimientos especializados en los campos en los que se utilizan los
+paquetes.
+
+Por favor también lee [nuestra guía para personas que realizan
+revisiones](https://devdevguide.netlify.app/es/softwarereview_reviewer.es.html).
+
+## <a href="#further" name="further"></a>Más recursos
+
+Nuestra [Guía de
+desarrollo](https://devdevguide.netlify.app/es/index.es.html) también
+inlcuye:
+
+- [una guía para el equipo
+  editorial](https://devdevguide.netlify.app/es/softwarereview_editor.es.html)
+
+- [nuestras
+  políticas](https://devdevguide.netlify.app/es/softwarereview_policies.es.html)
+
+- capítulos dedicados a buenas prácticas para el mantenimiento de
+  paquetes
+
+- diferentes plantillas: [plantilla de
+  revisión](https://devdevguide.netlify.app/es/reviewtemplate.es.html),
+  [plantilla para quien hace la
+  edición](https://devdevguide.netlify.app/es/editortemplate.es.html),
+  [plantilla para pedido de
+  revisión](https://devdevguide.netlify.app/es/reviewrequesttemplate.es.html).
+
+Nuestro proceso de revisión está en constante desarrollo de modo que
+promovemos las devoluciones y las discusiones sobre cómo mejorarlo en
+nuestro [foro](https://discuss.ropensci.org/) y en [los issues del
+repositorio
+ropensci/software-review-meta](https://github.com/ropensci/software-review-meta/issues).
+
+# <a href="#editors" name="editors"></a>Equipo editorial y de revisión
+
+### Editores/as asociados/as
+
+El proceso de revisión por pares del software de rOpenSci está dirigido
+por:
+
+- [Noam Ross](https://github.com/noamross), EcoHealth Alliance, EEUU;
+- [Karthik Ram](https://github.com/karthik), rOpenSci, EEUU;
+- [Maëlle Salmon](https://github.com/maelle), rOpenSci, Francia;
+- [Anna Krystalli](https://github.com/annakrystalli), University of
+  Sheffield RSE, Reino Unido;
+- [Mauro Lepore](https://github.com/maurolepore), 2 Degrees Investing
+  Initiative, EEUU;
+- [Laura DeCicco](https://github.com/ldecicco-USGS), USGS, EEUU;
+- [Julia Gustavsen](https://github.com/jooolia), Agroscope, Suiza;
+- [Emily Riederer](https://github.com/emilyriederer), Capital One, EEUU;
+- [Adam Sparks](https://github.com/adamhsparks), Department of Primary
+  Industries and Regional Development;
+- [Jeff Hollister](https://github.com/jhollist), US Environmental
+  Protection Agency.
+
```

Comment:
I think we need to update this list. For example, Bea is not here.

Reply:
it will be knit again automatically but i'll check